### ISSUE TITLE
`go install github.com/Yash-Handa/logo-ls@latest` fails

### ISSUE THREAD

**Issue by brumhard** [Yash-Handa/logo-ls#26]
Hi there,

So I was trying to install logo-ls on my machine.
Since go1.16 `go install <package>@latest` is the default why to install go tools from a remote repo (see [here](https://golang.org/doc/go1.16#modules)).
When trying to do that with your project I get the following error:
```shell
❯ go install github.com/Yash-Handa/logo-ls@latest
go: downloading github.com/Yash-Handa/logo-ls v1.3.7
go install github.com/Yash-Handa/logo-ls@latest: create zip: testdata/logo-ls-a.snap: case-insensitive file name collision: "testdata/logo-ls-A.snap" and "testdata/logo-ls-a.snap"
testdata/logo-ls-r.snap: case-insensitive file name collision: "testdata/logo-ls-R.snap" and "testdata/logo-ls-r.snap"
```

I guess the issue comes from zip not being case sensitive. It would be great if you could rename the test files to allow the `go install` way since that is much easier than cloning & building manually as long as homebrew support is not there.

**Comment by brumhard** [comment]
Also I couldn't even clone the repo properly on mac, since the file name collisions lead to an error. I guess that is only working on linux then.

**Comment by brumhard** [comment]
I've created a fork to enable the `go install` for me. You can check it out with `go install github.com/brumhard/logo-ls@0d88fb0b986e5a65eb65af14a79c52d8631e31c6` (since latest does not work without a new tag).
I had to add some code to fix the tests for mac. I'd be happy to contribute that back to your repo with a PR if you accept it.

**Comment by wasi-master** [comment]
Same issue here

**Comment by brumhard** [comment]
@wasi-master yeah well sadly @Yash-Handa hasn't replied so I guess this won't be fixed anymore 👀
